test/set.js

```diff
@@ -25,9 +25,10 @@ assert.equal(result.stderr, 'ls: no such file or directory: file_doesnt_exist\n'
 
 // set -e
 var result = shell.exec('node -e \"require(\'../global\'); set(\'-e\'); ls(\'file_doesnt_exist\'); echo(1234);\"');
-assert.equal(result.code, 1);
+var uncaughtErrorExitCode = process.version.indexOf('v0.10') === 0 ? 8 : 1;
```

Comment:
Is this only for version v0.10, or is this for all versions earlier than v0.11? If it's the latter, a `< 'v0.11'` might be better